easybuild/easyconfigs/b/Bazel/Bazel-0.29.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Bazel'
+version = '0.29.1'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/bazelbuild/bazel/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+patches = ['Bazel-%(version)s_fix-gold-flag.patch']
+checksums = [
+    '872a52cff208676e1169b3e1cae71b1fe572c4109cbd66eab107d8607c378de5',  # bazel-0.29.1-dist.zip
+    '99928d0902beeaf962a8ad14db8432f8e5114645e3caf64c7ee2fa136c31609f',  # Bazel-0.29.1_fix-gold-flag.patch
+]
+
+builddependencies = [
+  ('binutils', '2.31.1'),
+  ('Python', '3.7.2'),
+]
+dependencies = [('Java', '11', '', True)]
```

Comment:
You'll need to stick to `Java` 1.8 here to make the test suite pass (we stick to a single dependency version per toolchain generation, and `Java` 1.8 is already used in other easyconfigs that use `GCCcore/8.2.0` as toolchain)

```python
dependencies = [('Java', '1.8', '', True)]
```